app/pods/publications/controller.js

```diff
@@ -48,15 +48,17 @@ export default class PublicationsController extends Controller {
 
   @action
   async search() {
-    const filter = {};
+    const filter = {
+      ':has:publicationFlowNumber': 1,
+    };
     if (this.searchText.length === 0 || this.searchText === '') {
       filter[':sqs:title'] = '*'; // search without filter
       this.showSearchResults = false;
```

Comment:
Een zoekopdracht runnen terwijl je ze niet toont is wat contra-intuïtief. Als er een reden voor is kan 't blijven hé ...

Reply:
Inderdaad, aangepast